store/types/store.go

```diff
@@ -22,6 +22,9 @@ type Committer interface {
 	Commit() CommitID
 	LastCommitID() CommitID
 
+	// WorkingHash returns the hash of the KVStore's state before commit.
+	WorkingHash() []byte
```

Comment:
why not have error here?

Reply:
most functions are using panic inside it, do you think you need to refactor it?

Reply:
we can keep for now.